This file is part of MXE.
See index.html for further information.

From d307d2b671c5bc1e3bab850dcb8c3450fee62f66 Mon Sep 17 00:00:00 2001
From: Timothy Gu <timothygu99@gmail.com>
Date: Fri, 27 Jun 2014 11:58:14 -0700
Subject: [PATCH] Build import libs as .dll.a instead of .lib

This is MinGW convention.

Signed-off-by: Timothy Gu <timothygu99@gmail.com>
---
 Makefile.install | 2 +-
 exports/Makefile | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.install b/Makefile.install
index 9fc8d7a..7eb7b5f 100644
--- a/Makefile.install
+++ b/Makefile.install
@@ -83,7 +83,7 @@ ifeq ($(OSNAME), Darwin)
 endif
 ifeq ($(OSNAME), WINNT)
 	@-cp $(LIBDLLNAME) $(OPENBLAS_BINARY_DIR)
-	@-cp $(LIBPREFIX).lib $(OPENBLAS_LIBRARY_DIR)
+	@-cp $(LIBDLLNAME).a $(OPENBLAS_LIBRARY_DIR)
 endif
 ifeq ($(OSNAME), CYGWIN_NT)
 	@-cp $(LIBDLLNAME) $(OPENBLAS_BINARY_DIR)
diff --git a/exports/Makefile b/exports/Makefile
index 3ef20b3..ce28ab5 100644
--- a/exports/Makefile
+++ b/exports/Makefile
@@ -84,7 +84,7 @@ dll  : ../$(LIBDLLNAME)
 ../$(LIBDLLNAME) : ../$(LIBNAME) libopenblas.def dllinit.$(SUFFIX)
 	$(RANLIB) ../$(LIBNAME)
 	$(CC) $(CFLAGS) $(LDFLAGS) libopenblas.def dllinit.$(SUFFIX) \
-	-shared -o ../$(LIBDLLNAME) -Wl,--out-implib,../$(LIBPREFIX).lib \
+	-shared -o ../$(LIBDLLNAME) -Wl,--out-implib,../$(LIBDLLNAME).a \
 	-Wl,--whole-archive ../$(LIBNAME) -Wl,--no-whole-archive $(FEXTRALIB)
 
 libopenblas.def : gensymbol
-- 
1.9.1

